### ISSUE TITLE
PRO-not mac compatible

### ISSUE THREAD

**Issue by maxx270** [igdmapps/igdm#1313]
Hello,

I purchased the pro version but cannot use it due to malicious software.

Thanks,

Trevor

**Comment by EpicGamesOfficial** [comment]
Hi Trevor,
What do you mean you cannot use the pro version due to malicious software?

**Comment by maxx270** [comment]
When I attempt to run it my Mac says I can't due to it possibly containing malware

**Comment by EpicGamesOfficial** [comment]
Is this chrome doing this?

**Comment by maxx270** [comment]
Nope, I've downloaded it and installed, just tried to run it and macos is saying it can't verify it's not malicious

**Comment by EpicGamesOfficial** [comment]
Try this,
https://www.jamf.com/jamf-nation/discussions/34092/catalina-verifying-application-name-first-time-users-run-application-after-login
